src/tests/operate.test.js

```diff
@@ -0,0 +1,51 @@
+import operate from '../components/Calculator/operate.js';
+
+it('Test for the adding operation (2+2=4)', () => {
+  const mockState = {
+    total: '2',
+    next: '2',
+    operation: '+',
+  };
+
+  expect(operate(mockState.total, mockState.next, mockState.operation)).toBe('4');
+});
+
+it('Test for the minus operation (10-5=5)', () => {
+  const mockState = {
+    total: '10',
+    next: '5',
+    operation: '-',
+  };
+
+  expect(operate(mockState.total, mockState.next, mockState.operation)).toBe('5');
+});
+
+it('Test for the multiply operation (10x5=50)', () => {
+  const mockState = {
+    total: '10',
+    next: '5',
+    operation: 'x',
+  };
+
+  expect(operate(mockState.total, mockState.next, mockState.operation)).toBe('50');
+});
+
+it('Test for the division operation (10/2=5)', () => {
+  const mockState = {
+    total: '10',
+    next: '2',
+    operation: '÷',
+  };
+
+  expect(operate(mockState.total, mockState.next, mockState.operation)).toBe('5');
+});
+
+it('Test for the division operation (50%100=50)', () => {
+  const mockState = {
+    total: '50',
+    next: '100',
+    operation: '%',
+  };
+
+  expect(operate(mockState.total, mockState.next, mockState.operation)).toBe('50');
+});
```

Comment:
- [ ] Please start with `describe` to describe what is happening at next.

- [ ] Good job here, however, let us add more tests to move forward.

- Should throw an error if given a wrong number
- Should give a 0 error if given a wrong operator

Reply:
Thanks for your comment!
Please some of your suggestions don't have any sense, please try to check before submitting 
Example:
'what is happening at next'
'let us add more tests'

Reply:
Not sure what you mean by it doesn’t make sense, kindly take a look in this [resource](https://www.xoriant.com/blog/product-engineering/jest-fast-furious-unit-testing-framework.html), you will understand what I meant